packages/shared/src/components/cards/PostCard.spec.tsx

```diff
@@ -107,10 +117,23 @@ it('should call on bookmark click on bookmark button click', async () => {
   );
 });
 
+it('should not display publication date based on features flags', async () => {
+  act(async () => {
```

Comment:
Why do we need the `act` function here?

Reply:
I think some re-renders are triggered, and the tests were failing. After I applied the `act` to signify that rendering might take time, it worked.

Reply:
Ah, my procedure was just wrong, which makes the act necessary.